.github/workflows/php.yml

```
@@ -10,22 +10,22 @@ on:
 
 jobs:
   build:
-    strategy:      
+    strategy:
       matrix:
-        php-versions: ["7.2", "7.3", "7.4", "8.0"]
+        php-versions: ["7.2", "7.3", "7.4", "8.0", "8.1"]
         symfony-versions: ["~3.0", "~4.0", "~5.0"]
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v1
-      
+
     - name: Setup PHP
       uses: shivammathur/setup-php@v1
```

Ouch, this still used an old version of `setup-php`. 😖  `v2` is needed for PHP 8.1 support.

```suggestion
      uses: shivammathur/setup-php@v2
```